src/baselaplace/predicting.jl

```diff
@@ -73,36 +104,68 @@ predict(la, hcat(x...))
 ```
 """
 function predict(
-    la::AbstractLaplace, X::AbstractArray; link_approx=:probit, predict_proba::Bool=true
+    la::AbstractLaplace,
+    X::AbstractArray;
+    link_approx=:probit,
+    predict_proba::Bool=true,
+    ret_distr::Bool=false,
 )
-    fμ, fvar = glm_predictive_distribution(la, X)
+    normal_distr, fμ, fvar = glm_predictive_distribution(la, X)
 
     # Regression:
     if la.likelihood == :regression
-        return fμ, fvar
+        return normal_distr
     end
 
     # Classification:
     if la.likelihood == :classification
+        has_finaliser = has_softmax_or_sigmoid_final_layer(la.model)
```

Comment:
Could you double-check if we should have this check earlier (during construction of the `Laplace` object)? In other words, what currently happens if a user tries to fit LA on a network that has softmax/sigmoid in its final layer?

Reply:
@Rockdeldiablo have you double-checked this?

Reply:
I've just adjust #98 for now so we can merge this

Reply:
@pat-alt  i tried and i didn't see anything that suggest that there is a problem and i didn't see a softmax/sigmoid applied twice. but honestly i am not sure i completely understood the passages. the algorithm you used is different from the simple one reported in the book.